stepsfunction/handler.py

```diff
@@ -163,19 +178,186 @@ def ocr_page(event, context):
     See Callback Pattern at https://console.aws.amazon.com/states/home?region=us-east-1#/sampleProjects
     """
     LOG.info(f'event: {dumps(event)}')
-    activity_arn = os.environ.get('ACTIVITY_ARN', "")
+    LOG.info(f'boto3 version: {boto3.__version__} botocore version: {botocore.__version__}')
```

Comment:
Great idea!  Some "best practices" articles I've read say not to trust the versions of Boto that AWS includes because they may be old, or they may be updated at any time and change things. But for now, if things work, logging version to give us some info will help us.